src/main/app/idam/idamClient.ts

```diff
@@ -72,4 +72,9 @@ export class IdamClient {
         )
       })
   }
+
+  static invalidateSession (code: string): Promise<void> {
```

Comment:
Can you please add precondition to throw an error when jwt is blank? Something like:

```
   if (!jwt) {
      return Promise.reject(new Error('JWT is required'))
    }
```

At the same time please rename `code` to be `jwt` to be more clear.